src/my-flashcards.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import NoFlashcards from './no-flashcards'
+
+export default class MyFlashcards extends React.Component {
+  render() {
+    if (this.props.flashcards.length === 0) {
+      return (
+        <NoFlashcards takeToForm={this.props.takeToForm} />
+      )
+    }
+    else {
+      return (
+        <div className="container p-2">
```

Comment:
This is a little weird having two containers nested like this. Any particular reason?